src/app/game/tic-tac-toe/multi-player/page.js

```
@@ -174,10 +257,8 @@ const Game = () => {
                                     handleCreateGame();
                                     setOptionNewGame(true);
                                     setOptionSelected(true);
-                                }}>Create Game
-                                </button>
-                                <button className={styles.copyButton} onClick={() => setOptionSelected(true)}>Join Game
-                                </button>
+                                }} disabled={loading}>Create Game</button>
+                                <button className={styles.copyButton} onClick={() => setOptionSelected(true)} disabled={loading}>Join Game</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)